src/server/models/cacheable_queries/user.js

```diff
@@ -32,6 +32,55 @@ export async function userHasRole(userId, orgId, acceptableRoles) {
   }
 }
 
+async function getUserOrgInfo(userId, orgId) {
+  const userOrgKey = `texterorg-${userId}-${orgId}`
+  let orgInfo = await r.redis.getAsync(userOrgKey)
+  if (!orgInfo) {
+    const orgData = await r.knex('user_organization')
+      .join('organization', 'organization.id', 'user_organization.organization_id')
+      .where({
+        user_id: userId,
+        organization_id: orgId
+      })
+      .select('organization.id', 'organization.name')
+      .distinct()
+    await r.redis.set(userOrgKey, JSON.stringify(orgData[0]))
+    return orgData[0]
+  } else {
+    let parsedOrgInfo = JSON.parse(orgInfo)
+    return parsedOrgInfo
+  }
+}
+
+export async function userOrgsWithRole(role, userId) {
```

Comment:
can we rename this `userOrgs(userId, role)`, and have `role` as the second argument which will be optional (since I believe it is)